packages/yoastseo/src/worker/AnalysisWebWorker.js

```diff
@@ -1243,11 +1234,10 @@ export default class AnalysisWebWorker {
 				synonyms: this._relatedKeywords[ key ].synonyms,
 			} );
 
-			// Which combination of (tree) assessors and score aggregator to use.
+			// Which combination of (tree) assessors to use.
```

Comment:
... and we can also call the `this.assess` function with just a `Paper` and an `Assessor`.